dist/infragram3.js

```diff
@@ -0,0 +1,2213 @@
+ isVideo = false,isCamera=false; // Turns off camera feed
```

Comment:
Ah ok! Did you make changes to a copy of dist/infragram.js? So what may be better is to look where in /src/ you want to make the changes, and do them there... Then they'll generate a new dist/infragram.js when we run `grunt build`. Does that make sense? It compiles the code into /dist/. @cesswairimu or @TildaDares both have experience with this too! No worries we'll help if it doesn't make sense.

Reply:
yes the `grunt build` command makes sense, thank you.